db/migrate/20210310205504_create_organization_capacities.rb

```diff
@@ -0,0 +1,5 @@
+class CreateOrganizationCapacities < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
This is nice syntax, and honestly maybe the `rake analytics:create_views` system should use this! Not a blocker at all.